src/main/java/com/maxmind/minfraud/response/CreditCard.java

```diff
@@ -67,14 +81,49 @@ public Boolean isPrepaid() {
         return isPrepaid;
     }
 
+    /**
+     * @return The credit card type.
+     */
+    @JsonProperty("type")
+    public Type getType() {
+        return type;
+    }
+
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder("CreditCard{");
         sb.append("issuer=").append(this.issuer);
+        sb.append(", brand='").append(this.brand).append('\'');
         sb.append(", country='").append(this.country).append('\'');
         sb.append(", isIssuedInBillingAddressCountry=").append(this.isIssuedInBillingAddressCountry);
         sb.append(", isPrepaid=").append(this.isPrepaid);
+        sb.append(", type='").append(this.type).append('\'');
         sb.append('}');
         return sb.toString();
     }
+
+    /**
+     * The enumerated credit card types
+     */
+    public enum Type {
+        BLANK,
+        CHARGE,
+        CREDIT,
+        DEBIT;
```

Comment:
If we are going to have an enum here, we should have a test that ensure a new value in the response does not break it.